### ISSUE TITLE
A white line appears below the mainwindow

### ISSUE THREAD

**Issue by CoderTyn** [microsoft/terminal#3650]
# Environment

```none
Windows build number: [run `[Environment]::OSVersion` for powershell, or `ver` for cmd]
Windows Terminal version (if applicable):
10.0.19025.1051
Any other software?
none
```

# Steps to reproduce
there's a white line below this window , no matter when I use dark theme or light theme
![image](https://user-images.githubusercontent.com/46312580/69301796-6b950000-0c52-11ea-985d-6aff0a7f9017.png)

# Expected behavior
it is a small bug... i hope it can be fixed in next version... thx

# Actual behavior

**Comment by j4james** [comment]
Sounds like a duplicate of #3064.

**Comment by zadjii-msft** [comment]
Thanks for the triage @j4james! Yep, this is definitely a /dup of #3064, which is fixed by #3394.
